.github/workflows/build.yaml

```diff
@@ -70,12 +70,29 @@ jobs:
           profile: minimal
           override: true
 
-      - name: Run tests
+      - name: Install Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          working-directory: integration_tests
+          ruby-version: 2.7.4
+          bundler-cache: true
+
+      - name: Run unit tests
         uses: actions-rs/cargo@v1
         with:
           command: test
           args: --features=mocks
 
+      - name: Start Mosquitto
```

Comment:
Not really sure why the `docker run` approach isn't working inside github actions, and it's painful to debug.

Reply:
I think this is fine; it's like sharing a postgres database, which is common in tests.

It's occurred to me that I could achieve multitenancy (and even test parallelisation) by applying a custom MQTT prefix (i.e. topics would be `lxp_test_XXXX/...`) in lxp-bridge's config. Will look into that.